src/frontend/src/env/tokens.btc.env.ts

```diff
@@ -51,3 +52,9 @@ export const BTC_REGTEST_TOKEN: Token = {
 	decimals: BTC_DECIMALS,
 	icon: bitcoinTestnet
 };
+
+export const SUPPORTED_BITCOIN_TOKENS: [...Token[], Token] = [
```

Comment:
Maybe add a comment (or a test) that the order matters here.

Reply:
done!